metadata.json

```diff
@@ -24,7 +24,8 @@
       "operatingsystem": "RedHat",
       "operatingsystemrelease": [
         "6",
-        "7"
+        "7",
+        "8"
```

Comment:
These two commits seem to be very different. Suggest sending a separate PR for EL8 support.

Reply:
@ghoneycutt I've tested this module on EL8 successfully, only necessitating a metadata update for compatibility. Would it really make sense to have a dedicated PR just to update the one line?